templates/register.html

```diff
@@ -0,0 +1,27 @@
+{%extends "base.html" %}
+
+{% block content %}
+<h1>Login</h1>
+<form method="POST" action="{{ url_for('/') }}">
+    {{form.csrf_token}}
+    {% for field in form if field.name != 'csrf_token' %}
+    <p>
+        {{ field.label }}<br>
+        {{ field }}
+        {% if field.errors %}
+    <ul class="alert alert-danger">
+        {% for error in field.errors %}
+        <li>{{ error }}</li>
+        {%endfor %}
+    </ul>
+    {% endif %}
+    </p>
+    {% endfor %}
+    <p>
+        {%endfor%}
+    </p>
+    <p>
+        <input type="submit" value="Register">
+    </p>
```

Comment:
Так ведь кнопка уже есть внутри формы и отдельно создавать ее не нужно
```suggestion
    <p>
        {{ form.submit() }}
    </p>
```